bootstrap-joining-demo/aws-api/ec2-tag-based/src/it/resources/application.conf

```diff
@@ -0,0 +1,6 @@
+akka {
+  log-config-on-start = "off"
+  actor {
+    provider = "local"
```

Comment:
shouldn't it be "cluster"?

Reply:
@patriknw the integration test just uses Akka Http to make some calls to AWS, so I turned off the cluster ;-(

Reply:
@patriknw Made some changes (replaced asserts with should matchers etc.). Let me know if you think there's anything else that requires changes.

Reply:
Thanks for adding the comment below